### DISCUSSION TITLE
[Design proposal] Cross filters scoping

### DISCUSSION THREAD

**Discussion by mihir174** [apache/superset#14270] [Show and tell]
We need a way to enable and disable charts from receiving and emitting cross filters.
The chart overflow menu is a good place for this since it's where other chart-related actions on the dashboard are configured, and it's easily accessible outside of edit mode.

The designs show:
- a re-organized chart overflow menu
- an added "Cross filters" item that allows users to toggle on/off a chart receiving/emitting cross filter events, and scoping emitted events

https://user-images.githubusercontent.com/64227069/114678894-8a20ee80-9cc0-11eb-8186-2311da93cf4c.mov

There is already a checkbox for charts in Explore, that enables or disables emitting cross filter events. The state of the toggle in the chart's overflow menu on the dashboard can reflect the state selected in Explore.

**Comment by amitmiran137** [comment]
we should start calling it **interactive actions** bc there are more to that then just filters
it could transmit a groupBy/ drilldown/ highlight  and whatever your mind can think of

**Comment by junlincc** [comment]
Dashboard interactive action is a broader term to include different filtering, and deep drill features. Let's keep X-filtering for this proposal. 
I think we have reached the lazy consensus and work has already started, so im converting it to Discussion.
